src/azure.rs

```diff
@@ -86,6 +162,16 @@ impl std::fmt::Display for MicrosoftAzure {
     }
 }
 
+#[allow(clippy::borrowed_box)]
```

Comment:
clippy wants to ro remove the `Box<>` in the function signature, but if I do it no longer compiles ...

Reply:
Did you change the callsites from `check_err_not_found(&err)` to `check_err_not_found(err.as_ref())`?